api/rbdigital.py

```diff
@@ -2039,7 +2044,9 @@ def __init__(self, _db, collection, api_class=RBDigitalAPI,
                  api_class_kwargs={}):
         """Constructor."""
         super(RBDigitalSyncMonitor, self).__init__(_db, collection)
-        self.api = api_class(_db, collection, **api_class_kwargs)
+        if not isinstance(api_class, RBDigitalAPI):
+            api_class = api_class(_db, collection, **api_class_kwargs)
+        self.api = api_class
```

Comment:
This is a common pattern with other vendor APIs; it lets me pass in an already-instantiated mock API class.